src/Smalot/PdfParser/PDFObject.php

```diff
@@ -92,7 +92,7 @@ public function getDocument(): Document
         return $this->document;
     }
 
-    public function getHeader(): ?Header
```

Comment:
That is an API change and would require us to raise major version of the library. Can you please revert it?